tuf/api/keys.py

```diff
@@ -0,0 +1,169 @@
+# Borrowed from https://github.com/cnabio/signy/blob/afba301697df456b363790dc16483408b626a8af/scripts/in-toto/keys.py
+# TODO:
+# * Make a storage/provider-agnostic (e.g., filesystem, HSM) key management API, like securesystemslib.storage.
+
+# Imports.
+
+# 1st-party.
+import os
+import shutil
+
+# 2nd-party.
+from typing import Any, Dict, List, Optional
+
+# 3rd-party.
+from securesystemslib.interface import (
+    generate_and_write_ed25519_keypair,
+    get_password,
+    import_ed25519_privatekey_from_file,
+    import_ed25519_publickey_from_file,
+)
+
+# Utility classes.
+
+class Threshold:
+
+    def __init__(self, m: int = 1, n: int = 1):
```

Comment:
Tiny nit, but can we use more descriptive parameter names here? I know it's an m-of-n scheme but `minimum` and `keyholders` (or similar) would be a little easier to read 🙂